File: replace-colors-scenarios.py
import re

file_path = r'C:\Users\DELL\Desktop\mortious\LexAI\src\pages\Scenarios.jsx'

# Read file
with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

# Perform replacements
replacements = [
    ('text-gray-400', 'text-muted-foreground'),
    ('text-gray-300', 'text-foreground'),
    ('text-gray-500', 'text-muted-foreground'),
    ('text-gray-600', 'text-muted-foreground'),
    ('text-dark-foreground', 'text-foreground'),
    ('text-dark-primary', 'text-foreground'),
    ('placeholder-gray-500', 'placeholder:text-muted-foreground'),
    ('hover:text-dark-foreground', 'hover:text-foreground'),
    ('hover:text-dark-primary', 'hover:text-foreground'),
    ('hover:ring-dark-primary/20', 'hover:ring-foreground/20'),
    ('focus:ring-dark-primary/50', 'focus:ring-foreground/50'),
    ('ring-dark-primary/20', 'ring-foreground/20'),
]

for old, new in replacements:
    content = content.replace(old, new)

# Write file
with open(file_path, 'w', encoding='utf-8') as f:
    f.write(content)

print('✓ All color classes replaced successfully')

File: replace-colors-scenarios.js
const fs = require('fs');

const filePath = 'C:\\Users\\DELL\\Desktop\\mortious\\LexAI\\src\\pages\\Scenarios.jsx';

// Read file
let content = fs.readFileSync(filePath, 'utf8');

// Perform replacements
content = content.replace(/text-gray-400/g, 'text-muted-foreground');
content = content.replace(/text-gray-300/g, 'text-foreground');
content = content.replace(/text-gray-500/g, 'text-muted-foreground');
content = content.replace(/text-gray-600/g, 'text-muted-foreground');
content = content.replace(/text-dark-foreground/g, 'text-foreground');
content = content.replace(/text-dark-primary/g, 'text-foreground');
content = content.replace(/placeholder-gray-500/g, 'placeholder:text-muted-foreground');
content = content.replace(/hover:text-dark-foreground/g, 'hover:text-foreground');
content = content.replace(/hover:text-dark-primary/g, 'hover:text-foreground');
content = content.replace(/hover:ring-dark-primary\/20/g, 'hover:ring-foreground/20');
content = content.replace(/focus:ring-dark-primary\/50/g, 'focus:ring-foreground/50');
content = content.replace(/ring-dark-primary\/20/g, 'ring-foreground/20');

// Write file
fs.writeFileSync(filePath, content, 'utf8');

console.log('✓ All color classes replaced successfully');
